AndelaOSP/wire
# fix: Unregistered login bug

#### What does this PR do?
Whenever a user who is not in the db tries to login, the login page would show a continuous spinner
and logs this error in the console:
`Uncaught (in promise) TypeError: Cannot read property 'status' of undefined at errorAction`

This PR fixes that.
#### Where should the reviewer start?
View file changes in PR
#### How should this be manually tested?
Using either the inbuilt mock API or localhost API, try logging in as an unregistered user.
The login page should display this error message:

<img width="1680" alt="screen shot 2018-06-19 at 16 22 01" src="https://user-images.githubusercontent.com/25130454/41599834-f5f38c04-73dc-11e8-9b16-f199f990b54a.png">
